common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerRequest.java

```diff
@@ -253,4 +253,8 @@ private static final class SerializedNames {
     @Nullable
     @SerializedName(SerializedNames.PREFERRED_AUTH_METHOD)
     private PreferredAuthMethod mPreferredAuthMethod;
+
+    @Nullable
+    @SerializedName(SerializedNames.ACCOUNT_TRANSFER_TOKEN)
+    private String mAccountTransferToken;
```

Comment:
sync with @mohitc1.

If he's planning to deserialize the BrokerRequest in AccountChooser.shouldDisplayAccountList(), then you might not need this. you can use BrokerInteractiveTokenCommandParameters.isAccountTransferRequest() instead.

Reply:
My change was to allow the check without deserializing, Is it possible to do something similar in your implementation @mohitc1 ? Or do we feel we should pivot to deserializing the brokerRequest inside the account chooser.

Reply:
I pulled mohit's changes and used the earlier serializing infra